edu.princeton.physics.WSJTX.metainfo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>edu.princeton.physics.WSJTX</id>
+    <name>wsjtx</name>
```

Comment:
```suggestion
    <name>wsjtx</name>
    <provides>
        <id>wsjtx.desktop</id>
    </provides>
```

(So GNOME Software won't display this Flatpak as a separate entry, but as another source for the regular distribution package.)